src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,30 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
+      </h3>
+
+      <p>
+        {' Posted by  '}
+
+        ?
+        {' '}
+        <UserInfo user={post.user} />
+        :
+        {' '}
+        <p>user does not exist</p>
```

Comment:
```suggestion
{post.user && <UserInfo user={post.user} />}
```

And in this case we will see `UserInfo` component only if we have `post.user`